linty_fresh/reporters/github_reporter.py

```diff
@@ -189,34 +226,70 @@ def _get_issue_url(self) -> str:
                     repo=self.repo,
                     pr=self.pr))
 
-    async def get_existing_messages(
-        self, client_session: aiohttp.ClientSession
+    def _get_delete_pr_comment_url(self, id: int) -> str:
```

Comment:
`comment_id` maybe? `id` is a builtin function and this could cause some nightmare in the future. Been there.

Reply:
done